arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -1982,6 +1982,17 @@ void RocksDBEngine::addSystemDatabase() {
   builder.add("id", VPackValue(std::to_string(id)));
   builder.add("name", VPackValue(StaticStrings::SystemDatabase));
   builder.add("deleted", VPackValue(false));
+
+  //one shard
+  builder.add(StaticStrings::Sharding, VPackValue(std::string{}));
```

Comment:
Can we reuse the already existing method above here to append the sharding info and the replicationFactor to the builder? This would be good to have just one implementation of the logic. This method here is already different (doesn't pick up value of `--cluster.default-replication-factor`) than the other implementation.
So please use one implementation only, and that needs to do the right thing (i.e. pick up the correct defaults etc.).